libs/wingsdk/src/testing/simulator.ts

```diff
@@ -10,14 +9,14 @@ import { DefaultSimulatorFactory } from "../target-sim/factory.inflight";
 // eslint-disable-next-line import/no-restricted-paths
 import { Function } from "../target-sim/function.inflight";
 import { BaseResourceSchema, WingSimulatorSchema } from "../target-sim/schema";
-import { mkdtemp, readJsonSync } from "../util";
+import { readJsonSync } from "../util";
 
 /**
  * Props for `Simulator`.
  */
 export interface SimulatorProps {
   /**
-   * Path to a Wing simulator file (.wsim).
+   * Path to a Wing simulator output directory (.wsim).
    */
   readonly simfile: string;
```

Comment:
I would fully commit to the breaking change, we are pre-beta
```suggestion
   * Path to a Wing simulator output directory (.wsim).
   */
  readonly simdir: string;
```

Reply:
I'll keep it for now.